docs/api/qiskit-ibm-runtime/qiskit_ibm_runtime.Estimator.md

```diff
@@ -73,13 +70,9 @@ Initializes the Estimator primitive.
 
 ## Attributes
 
-<span id="estimator-circuits" />
```

Comment:
It seems fine to remove these. I'm not sure where we were getting them from before. I imagine the old Sphinx output - I don't think our script was dynamically computing the name `estimator-circuits`.

Reply:
is there any specific css set with this id?

Reply:
No, there shouldn't be. The reason we had these `<span>`s is solely for `#` anchors. I couldn't find anywhere using these anchors in production, so removing the link seems fine.